lib/widgets/layouts/adaptive_layout/adaptive_scaffold_view.dart

```diff
@@ -188,15 +191,22 @@ class AppScaffoldView extends StatelessWidget {
   List<NavigationDestination> getNavigationDestinations(BuildContext context) {
     return [
       NavigationDestination(
-        icon: const Icon(Icons.contacts_outlined),
+        icon: const TwakeNavigationIcon(
+          icon: Icons.contacts_outlined,
+          notificationCount: 0,
```

Comment:
It's default `notificationCount = 0`. Remove it

Reply:
you're right